.github/workflows/juntagrico-ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: ["3.8", "3.10"]
```

Comment:
Base juntagrico does not officially support 3.10 yet. But will "soon". So I'm fine leaving this.

Reply:
Great! Thank you! :-)